SmartDeviceLink/SDLProxy.m

```diff
@@ -416,17 +413,19 @@ - (void)handleSystemRequest:(NSDictionary *)dict {
     [SDLDebugTool logInfo:@"OnSystemRequest (notification)" withType:SDLDebugType_RPC toOutput:SDLDebugOutput_All toGroup:self.debugConsoleGroupName];
 
     SDLOnSystemRequest *systemRequest = [[SDLOnSystemRequest alloc] initWithDictionary:[dict mutableCopy]];
-    SDLRequestType *requestType = systemRequest.requestType;
-
-    // Handle the various OnSystemRequest types
-    if (requestType == [SDLRequestType PROPRIETARY]) {
-        [self handleSystemRequestProprietary:systemRequest];
-    } else if (requestType == [SDLRequestType LOCK_SCREEN_ICON_URL]) {
-        [self handleSystemRequestLockScreenIconURL:systemRequest];
-    } else if (requestType == [SDLRequestType HTTP]) {
-        [self sdl_handleSystemRequestHTTP:systemRequest];
-    } else if (requestType == [SDLRequestType LAUNCH_APP]) {
-        [self sdl_handleSystemRequestLaunchApp:systemRequest];
+    if (systemRequest != nil) {
```

Comment:
This should never return nil. An assert would be better here. Have you seen this happen?